test/db/cmd/cmd_pdr

```diff
@@ -28,8 +28,8 @@ e anal.gp=0xabc
 pdr~addu t0[7]
 EOF
 EXPECT=<<EOF
-0x17b8
-0x17c8
+0x17bc
```

Comment:
Is this new output correct ?

Reply:
Yes, GP is now 16-byte aligned when known, so GP-relative addresses moved +4 in pdr